spec/request/preneeds/branches_of_services_request_spec.rb

```diff
@@ -3,6 +3,7 @@
 
 RSpec.describe 'Branches of Service Integration', type: :request do
   include SchemaMatchers
+  before(:each) { Redis.current.flushall }
```

Comment:
I would test this in the FormFill initializer and only stub it or use a factory here.